charters/DataStore/charter.md

```diff
@@ -69,16 +69,17 @@ The objective of the Data Store group is to deliver a versioned immutable object
 
 ## Communication
 ### Slack Channels
```

Comment:
Could you cut and paste this into your Slack Channels section:

```
[HumanCellAtlas/data-store](https://humancellatlas.slack.com/messages/data-store): Discuss design specs and prototypes for the replicated data store
[HumanCellAtlas/data-store-eng](https://humancellatlas.slack.com/messages/data-store-eng): Discussions regarding ongoing data store development efforts

```
Thanks for updating.